spec/menu_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) {described_class.new}
+  let(:default_menu) { {ribs: 3,
+    beef: 4,
+    rolls: 3,
+    chips: 2,
+    pies: 5}}
+
+    describe '#show' do
+      it 'returns a menu' do
+        expect(menu.show).to eq default_menu
+      end
+    end
+  end
```

Comment:
Lint/BlockAlignment: `end` at 16, 2 is not aligned with `describe Menu do` at 3, 0.